stmt/stmt_test.go

```diff
@@ -254,6 +254,29 @@ func TestNextResetState(t *testing.T) {
 	}
 }
 
+func TestEmptyVariablesRawString(t *testing.T) {
+	stmt := new(Stmt)
+	stmt.Buf = []rune("select ")
```

Comment:
A unit test should not be setting this manually. Use `Append`, which is what's used internally on the call back from `rline`.